server/views/pages/crmDetails.njk

```diff
@@ -4,46 +4,50 @@
 {% from "../components/timeAndCost.njk" import timeAndCost %}
 {% from "../components/totalAndCost.njk" import totalAndCost %}
 
-{% set pageTitle = applicationName + " - Home" %}
+{% set pageTitle = applicationName + " - " + crmType + ": " + title %}
 {% set mainClasses = "app-container govuk-body" %}
 
 {% block content %}
 
     <div class="govuk-width-container">
         <main class="govuk-main-wrapper">
+            <span class="govuk-caption-l govuk-!-padding-bottom-1">eForm: {{ crmType }} | Case number: {{ usn }}</span>
             <h1 class="govuk-heading-l">{{ title }}</h1>
             <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
+                <div class="govuk-grid-column-one-third ">
```

Comment:
You can delete this `div` here as it is already existing in the component or vice-versa, deleting it in the component if you prefer to have it here.

Reply:
I will leave it here as it hasn't been included in the component